### PR TITLE
Tempo CLI: Add a simple command to convert parquet schemas

### PR DESCRIPTION
**Pull request by joe-elliott**
**What this PR does**:
- Adds a command `tempo-cli util convert` that will convert a parquet data file from one schema to another.

### CODE REVIEW

File: cmd/tempo-cli/cmd-convert-block.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	}
+	sch := parquet.SchemaOf(new(vparquet.Trace))
+	writer := parquet.NewWriter(fOut, sch)
+
+	conversion, err := parquet.Convert(sch, pf.Schema())
+	if err != nil {
+		return err
+	}
+
+	// copy a rowgroup at a time
```
Comment (art-licis) [inline cmd/tempo-cli/cmd-convert-block.go:43]: Would it make sense to add some output to command execution, possibly progress over `len(pf.RowGroups)` processed? (Percentage or 'x out of n processed' - based on some time interval).

Comment (joe-elliott) [reply cmd/tempo-cli/cmd-convert-block.go:43]: Thanks for the suggestion. Added some progress messages.

File: cmd/tempo-cli/cmd-convert-block.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import (
+	"fmt"
+	"os"
+
+	"github.com/grafana/tempo/tempodb/encoding/vparquet"
+	"github.com/segmentio/parquet-go"
+)
+
+type convertParquet struct {
+	In  string `arg:"" help:"The input parqruet file to read from"`
```
Comment (mapno) [inline cmd/tempo-cli/cmd-convert-block.go:12]:
```suggestion
	In  string `arg:"" help:"The input parquet file to read from"`
```

**Review by mapno — APPROVED** [review]
LGTM

File: docs/tempo/website/operations/tempo_cli.md [hunk 3]
```diff
@@ -257,3 +257,19 @@ See backend options above.
 ```bash
 tempo-cli search blocks http.post GET 2021-09-21T00:00:00 2021-09-21T00:05:00 single-tenant --backend=gcs --bucket=tempo-trace-data
 ```
+
+## Parquet convert command
+Converts a parquet file from its existing schema to the one currently in the repo. This utility command is useful when
```
Comment (knylander-grafana) [inline docs/tempo/website/operations/tempo_cli.md:262]:
```suggestion
Converts a parquet file from its existing schema to the one currently in the repository. This utility command is useful when
```

**Pull request merged**